Mathlib/Topology/ContinuousFunction/StoneWeierstrass.lean

```diff
@@ -3,8 +3,10 @@ Copyright (c) 2021 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison, Heather Macbeth
 -/
+import Mathlib.Algebra.Algebra.Subalgebra.Unitization
 import Mathlib.Analysis.RCLike.Basic
 import Mathlib.Topology.Algebra.StarSubalgebra
+import Mathlib.Topology.ContinuousFunction.ContinuousMapZero
 import Mathlib.Topology.ContinuousFunction.Weierstrass
```

Comment:
The `ContinuousMapZero` file is probably just that file itself. The `Unitization` file pulls in `Unitization` and all the non-unital subobject stuff. So that's a bit heavier, but in my mind it's not too bad.

Reply:
Yeah, I guess that's not too bad.